deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/ocnn/OCNNOutputLayerTest.java

```diff
@@ -139,6 +143,12 @@ public void testOutput() {
 
         INDArray output = network.output(arr.getRows(ArrayUtil.range(102,149)));
         System.out.println(output);
+        File tmpFile = new File("tmp-ocnn-zip");
+        tmpFile.deleteOnExit();
+        ModelSerializer.writeModel(network,tmpFile,true);
+
+        MultiLayerNetwork loaded = ModelSerializer.restoreMultiLayerNetwork(tmpFile);
+        assertNotNull(loaded);
```

Comment:
You can replace these 5 lines with a 1-line TestUtils.testModelSerialization(...) call
https://github.com/deeplearning4j/deeplearning4j/blob/b555e1ca4963159e8bb9bcea5c0c162819c7ee0f/deeplearning4j-core/src/test/java/org/deeplearning4j/TestUtils.java#L18